### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Fixes farming unlimited metal from holodeck racks. Fixes https://github.com/ParadiseSS13/Paradise/issues/2133.
2) Refactors implant pad/implant locator code. You can now remove implants from implant pads, and implant locators now show more (useful) information. Fixes https://github.com/ParadiseSS13/Paradise/issues/2122.
3) Adds the missing fingerless sprite for mobs. Fixes https://github.com/ParadiseSS13/Paradise/issues/2130.
4) Fixes the death ripley closed sprite. Fixes https://github.com/ParadiseSS13/Paradise/issues/2072.
5) Fixes research consoles not auto syncing. Fixes https://github.com/ParadiseSS13/Paradise/issues/2056. 
6) Admin teleporting to areas will no longer end you up in grilles/windows.
7) Access checks on research consoles now actually work, and fixes the computer not working properly after screwdrivering it. Fixes https://github.com/ParadiseSS13/Paradise/issues/2132.

### CODE REVIEW

File: code/game/machinery/computer/HolodeckControl.dm [hunk 1]
```diff
@@ -363,53 +345,24 @@
...
-		G.affecting.loc = src.loc
-		G.affecting.Weaken(5)
-		visible_message("\red [G.assailant] puts [G.affecting] on the table.")
-		qdel(W)
-		return
+	if (istype(W, /obj/item/weapon/grab))
+		return ..()
 
 	if (istype(W, /obj/item/weapon/wrench))
-		user << "It's a holotable!  There are no bolts!"
```
Comment (tigercat2000) [inline code/game/machinery/computer/HolodeckControl.dm]: Hm. I wonder why this check didn't function properly before.

Comment (Markolie) [inline code/game/machinery/computer/HolodeckControl.dm]: It did. Racks aren't a subtype of table though, so they needed a separate attackby proc.

File: code/game/objects/items/devices/radio/beacon.dm [hunk 2]
```diff
@@ -85,4 +92,5 @@
 		new /obj/machinery/telepad_cargo(user.loc)
 		playsound(src, 'sound/effects/pop.ogg', 100, 1, 1)
 		qdel(src)
-	return
\ No newline at end of file
+	return
+
```
Comment (tigercat2000) [inline code/game/objects/items/devices/radio/beacon.dm]: This file appears to end as git likes. But then has a tab. Which git doesn't like.

File: code/game/objects/items/weapons/implants/implant.dm [hunk 3]
```diff
@@ -72,24 +79,23 @@ disintegrate into bio-safe elements.<BR>
...
 
-		spawn(delay)
-			malfunction--
+	var/delay = 20
+	switch(severity)
+		if(1)
+			if(prob(60))
+				meltdown()
+		if(2)
+			delay = rand(5*60*10,15*60*10)	//from 5 to 15 minutes of free time
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/implants/implant.dm]: What is with this rand() proc? Why isn't it just 3000, 9000

Comment (Markolie) [inline code/game/objects/items/weapons/implants/implant.dm]: That's oldcode for you.

**Comment by Spacemanspark** [comment]
Noice.

**Comment by M3hillus** [comment]
Pssst. Did you include stools and tables?

**Comment by Markolie** [comment]
Those are already a-okay IIRC.

**Comment by Spacemanspark** [comment]
Well, I can't table anybody with them, so I'd assume they are.

**Pull request merged**